SingularityUI/app/components/common/ModalButton.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { Component, PropTypes } from 'react';
+import Modal from 'react-bootstrap/lib/Modal';
+import Button from 'react-bootstrap/lib/Button';
+
+export default class ModalButton extends Component {
```

Comment:
Based on what I think this is doing, ConfirmModal might be a better name than ModalButton